### ISSUE TITLE
Add skip button

### ISSUE THREAD

**Issue by ICEREG1992** [ICEREG1992/fungwafrenzy#19]
## Context

The original skip button appeared to the right of the player after a few seconds. Clicking it would take the user directly to the next video, or the next point where buttons are displayed.

**Comment by ICEREG1992** [comment]
This is technically addressed but is being blocked by other issues, namely #33. Closing in favor of that issue
